master/buildbot/data/builds.py

```diff
@@ -20,6 +20,31 @@
 
 class Db2DataMixin(object):
 
+    def _generate_filtered_properties(self, props, filters):
+        """
+        This method returns Build's properties according to property filters.
```

Comment:
I do not see the `rtype-builds.rst` file in the developer section.
Should I create it or do you mean `rtype-build.rst`?